src/formatters/__tests__/__fixtures__/oas3-schema-errors.json

```diff
@@ -1,7 +1,6 @@
 [
   {
     "code": "oas3-schema",
-    "summary": "Validate structure of OpenAPIv3 specification.",
```

Comment:
Why did this go away? Was summary deprecated, and this PR is removing it?

Reply:
>Why did this go away? Was summary deprecated, and this PR is removing it?

We removed `summary` some time ago. The fixture was just not updated at the time we did it.